soci/index_builder.go

```diff
@@ -31,26 +31,29 @@ import (
 )
 
 const (
-	DefaultSpanSize            = int64(1 << 22)
-	MinLayerSize               = 0
-	DefaultBuildToolIdentifier = "AWS SOCI CLI v0.1"
+	defaultSpanSize            = int64(1 << 22)
+	minLayerSize               = 0
```

Comment:
nit: We have updated this to 10 MiB elsewhere. Can you update it here too?

Reply:
I'm glad that we're having this PR discussion, otherwise this could be missed. I'll update it.